dissect/target/plugin.py

```diff
@@ -552,6 +559,7 @@ def register(plugincls: type[Plugin]) -> None:
                 exported=bool(len(exports)),
                 internal=bool(len(functions)) and not bool(len(exports)),
                 findable=plugincls.__findable__,
+                alias=getattr(attr, "__alias__", False),
```

Comment:
```suggestion
                alias=False,
```
I don't think namespaces themselves can be aliased, right?